concourse/pipeline.yml

```diff
@@ -20,4 +20,4 @@ jobs:
       - task: publish
         file: fauna-vscode-repository/concourse/tasks/publish.yml
         params:
-          VSCE_PAT: ((vscode-marketplace-token))
+          VSCE_PAT: ((vscode-marketplace-token_expires-07/23))
```

Comment:
Possible to replace 07/23 with YYYY_MM_DD

Reply:
Yeah definitely I'll do that